frontend/src/components/reflection/reflectionSubjectArea/reflectionSubjectArea.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+
+import {
+  IonList,
+  IonText,
+  IonItem,
+  IonLabel,
+  IonBadge,
+  IonIcon,
+  IonButton,
+} from '@ionic/react'
+
+import {
+  badgeStatus,
+  checkmarkIcon,
+  currentSelectedTopic,
+  doneStatusText,
+  textLable,
+} from './reflectionSubjectAreaUtils'
+
+interface TopicItem {
+  id: string
+  lable: string
+  describe: string
+  subListItems: SubTopicItem[]
+}
+
+export interface SubTopicItem {
+  id: string
+  lable: string
+  done: boolean
+}
+
+export type InputSelectValue = string | 'Später Wählen'
+
+interface Props {
+  selectValue: (input: InputSelectValue) => void
+  topicList: TopicItem[]
+}
+
+/*
+    The Component is designed to not store the selected value in the second level
+    You must Provide a Callback-Function where the value is emitted to and store
+    the state outside of this Component
+ */
+export const SubjectArea: React.FC<Props> = ({ topicList, selectValue }) => {
+  // this state stores the current selected row
+  // it is the id or undefined if nothing is selected
+  const [currentSelected, setCurrentSelected] = React.useState<
+    undefined | string
+  >(undefined)
+
+  const selectTopic = React.useCallback(
+    (
+      event:
+        | React.MouseEvent<HTMLIonItemElement>
+        | React.MouseEvent<HTMLIonButtonElement, MouseEvent>,
+    ) => {
+      //@ts-ignore
+      const selectedTopic: string = event.target.getAttribute('data-id')
+
+      setTimeout(() => {
+        selectValue(selectedTopic)
+      }, 300)
+    },
+    [selectValue],
+  )
+
+  // Set the currentSelected value to the clicked id
+  // if the id is Already selected, set the current selected value to undefined
+  // this Behavior is needed to unselect the row
+  const handlerCurrentSelected = (
+    event: React.MouseEvent<HTMLIonItemElement>,
+  ): void => {
+    //@ts-ignore
+    const clickedRow = event.target.getAttribute('data-id')
+    setCurrentSelected(currentSelected =>
+      currentSelected === clickedRow ? undefined : clickedRow,
+    )
+  }
+
+  const SubSubjectAreaList: React.FC<{ subListItems: SubTopicItem[] }> = ({
+    subListItems,
+  }) => {
+    return (
+      <IonList>
+        {subListItems.map(({ lable, done, id }, subIndex) => (
```

Comment:
```suggestion
        {subListItems.map(({ subjectLable, subjectStatus, id }, subIndex) => (
```
`done` is not very nice! Won't comment all the other places were `done` is used